docs/deployment/resource-providers/mesos.md

```diff
@@ -84,113 +77,127 @@ of failures.
-These files contain in each row a single hostname on which the respective component will be started (assuming SSH access to these nodes).
+[DC/OS examples documentation](https://github.com/dcos/examples/tree/master/flink).
 
-Next you have to create `MESOS_HOME/etc/mesos/mesos-master-env.sh` or use the template found in the same directory.
-In this file, you have to define
+#### Mesos without DC/OS
 
-    export MESOS_work_dir=WORK_DIRECTORY
+You can also run Mesos without DC/OS. Please follow the 
+[instructions on how to setup Mesos on the official website](http://mesos.apache.org/getting-started/).
 
-and it is recommended to uncommment
+After installing Mesos into `${MESOS_HOME}`, you have to configure the set of master and 
+agent nodes by creating the files `${MESOS_HOME}/etc/mesos/masters` and 
+`${MESOS_HOME}/etc/mesos/slaves`. These files contain a single hostname per row pointing to the 
+hosts executing the respective components (assuming SSH access being available for these nodes).  
 
-    export MESOS_log_dir=LOGGING_DIRECTORY
+Next, you have to create `${MESOS_HOME}/etc/mesos/mesos-master-env.sh` (a template file is 
+provided in the same directory). In `mesos-master-env.sh`, you have to set `${MESOS_work_dir}` 
+pointing to a directory used by Mesos to store data. Additionally, it is recommended to specify 
+`${MESOS_log_dir}`:
 
+{% highlight bash %}
+export MESOS_work_dir=/path/to/mesos/working/directory
+export MESOS_log_dir=/path/to/mesos/logging/directory
+{% endhighlight %}
 
-In order to configure the Mesos agents, you have to create `MESOS_HOME/etc/mesos/mesos-agent-env.sh` or use the template found in the same directory.
-You have to configure
-
-    export MESOS_master=MASTER_HOSTNAME:MASTER_PORT
-
-and uncomment
-
-    export MESOS_log_dir=LOGGING_DIRECTORY
-    export MESOS_work_dir=WORK_DIRECTORY
-
-#### Mesos Library
+The Mesos agents are configured by creating `${MESOS_HOME}/etc/mesos/mesos-agent-env.sh` (a 
+template file is provided in the same directory). `${MESOS_master}` needs to be configured as part 
+of `mesos-agent-env.sh`. Additionally, `${MESOS_work_dir}` and `${MESOS_log_dir}` should be set to 
+the corresponding folders:
 
-In order to run Java applications with Mesos you have to export `MESOS_NATIVE_JAVA_LIBRARY=MESOS_HOME/lib/libmesos.so` on Linux.
-Under Mac OS X you have to export `MESOS_NATIVE_JAVA_LIBRARY=MESOS_HOME/lib/libmesos.dylib`.
+{% highlight bash %}
+export MESOS_master=MASTER_HOSTNAME:MASTER_PORT
+export MESOS_work_dir=/path/to/mesos/working/directory
+export MESOS_log_dir=/path/to/mesos/logging/directory
+{% endhighlight %}
 
-#### Deploying Mesos
+In order to start your Mesos cluster, use the deployment script `${MESOS_HOME}/sbin/mesos-start-cluster.sh`.
+Stopping the Mesos cluster can be achieved through `${MESOS_HOME}/sbin/mesos-stop-cluster.sh`.
```

Comment:
Is this information maybe contained in the getting started guide for Mesos itself?

Reply:
You're right. I removed the information on how to install Mesos.